code/LeftAndMain.php

```diff
@@ -412,6 +412,21 @@ public function schema(HTTPRequest $request): HTTPResponse
         return $this->getSchemaResponse($schemaID, $form);
     }
 
+    /**
+     * Creates a successful json response
+     */
+    protected function jsonSuccess(int $statusCode, array $data = null): HTTPResponse
+    {
+        if ($statusCode < 200 || $statusCode >= 300) {
+            throw new InvalidArgumentException("Status code $statusCode must be between 200 and 299");
+        }
+        $body = is_null($data) ? '' : json_encode($data, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
```

Comment:
Why not use an empty array as the "no data" marker?

Reply:
Updated